packages/discovery-provider/src/api/v1/users.py

```diff
@@ -2670,3 +2674,61 @@ def get(self, id):
         )
         tracks = get_user_tracks_remixed(query_args)
         return success_response(list(map(extend_track, tracks)))
+
+
+USER_FEED_ROUTE = "/<string:id>/feed"
+user_feed_parser = current_user_parser.copy()
+user_feed_parser.add_argument(
+    "filter",
+    required=False,
+    type=str,
+    choices=["all", "repost", "original"],
+    default="all",
+)
+user_feed_parser.add_argument(
+    "tracks_only",
+    type=inputs.boolean,
+    required=False,
+)
+user_feed_parser.add_argument(
+    "with_users",
+    type=inputs.boolean,
+    required=False,
+)
+user_feed_parser.add_argument(
+    "followee_user_id", action="append", type=int, required=False
+)
+
+user_feed_response = make_full_response(
+    "user_feed_response", full_ns, fields.List(NestedOneOf(user_feed_item))
+)
+
+# TODO: log_duration, record_metrics
```

Comment:
Should this stay as a TODO for now?

Reply:
Let me see how long it would take to add it. FWIW, I don't think we track these kinds of things in other v1 endpoints. I just noticed that the root-level implementation in `queries` used some decorators for this.

Reply:
Turns out we _do_ use those decorators in v1, so added them to this endpoint as well